tests/admin.reported.comment.test.js

```diff
@@ -0,0 +1,65 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../src/index';
+import status from '../src/helpers/constants/status.codes';
+
+
+let userToken = '';
+
+const { expect } = chai;
+const password = 'password23423';
+
+chai.use(chaiHttp);
+
+describe('test the controller for an admin to get reported comment', () => {
+  it('should login the moderetor to get the token', (done) => {
+    chai
+      .request(app)
+      .post('/api/users/login')
+      .send({
+        email: 'alain13@gmail.com',
+        password
+      })
+      .end((err, res) => {
+        res.should.have.status(status.OK);
+        userToken = res.body.user.token;
+        done();
+      });
+  });
+
+  it('Admin should be able to get all reported comments ', (done) => {
+    chai.request(app)
+      .get('/api/admin/comments/reported')
+      .set('Authorization', userToken)
+      .end((err, res) => {
+        expect(res.body).to.have.property('status').eql(status.OK);
+        expect(res.body).to.have.property('paginationDetails');
+        expect(res.body.data).to.be.an('array');
+        done();
+      });
+  });
+
+  it('Admin should be able to get all reported comments and respond when there are any reports', (done) => {
+    const id = 4;
+    chai.request(app)
+      .get(`/api/admin/comments/${id}/reported`)
+      .set('Authorization', userToken)
+      .end((err, res) => {
+        expect(res.body).to.have.property('status').eql(status.OK);
+        expect(res.body).to.have.property('message').eql('No reports for this comment.');
+        done();
+      });
+  });
+
+  it('Admin should be able to get all reported comments and respond when there are any reports', (done) => {
+    const id = 6;
+    chai.request(app)
+      .get(`/api/admin/comments/${id}/reported`)
+      .set('Authorization', userToken)
+      .end((err, res) => {
+        expect(res.body).to.have.property('status').eql(status.OK);
+        expect(res.body.data).to.have.property('id');
+        done();
```

Comment:
@MCFrank16 also confirm here that actual data is returned.

Reply:
cool, lemme do it